doc/operation-and-maintenance/Logging-&-monitoring.md

```diff
@@ -84,60 +84,47 @@ By default, the resolution is set to 60 seconds.
 
-Route subnet:
-
-    $ sudo route add -net 172.17.0.0 192.168.99.100
-
-#### Verification
-
-And now http://172.17.0.2 should show a graphite page.
-
-Check if the data collection works - run:
-
-    $ while true; do echo -n "example:$((RANDOM % 100))|c" | nc -w 1 -u 172.17.0.2 8125; done
-
-Wait a while, then open:
+```
+Session count:                   <prefix>.global.totalSessionCount.value
+XMPP messages received:          <prefix>.<domain>.xmppMessageReceived.one
+XMPP messages sent:              <prefix>.<domain>.xmppMessageSent.one
+Successful logins:               <prefix>.<domain>.sessionSuccessfulLogins.one
+Logouts:                         <prefix>.<domain>.sessionLogouts.one
+Authorization time:              <prefix>.<domain>.backends.auth.authorize.<value-type>
+RDBMS "simple" query time:       <prefix>.<domain>.backends.mongoose_rdbms.query.<value-type>
+RDBMS prepared query time:       <prefix>.<domain>.backends.mongoose_rdbms.execute.<value-type>
+MAM lookups:                     <prefix>.<domain>.mam_lookup_messages.one
+MAM archivization time:          <prefix>.<domain>.backends.mod_mam.archive.<value-type>
+MAM lookup time:                 <prefix>.<domain>.backends.mod_mam.lookup.<value-type>
+MAM private messages flush time: <prefix>.<domain>.mod_mam_odbc_async_pool_writer.flush_time.<value-type>
+MAM MUC messages flush time:     <prefix>.<domain>.mod_mam_muc_odbc_async_pool_writer.flush_time.<value-type>
+```
 
-    http://172.17.0.2/render?from=-10mins&until=now&target=stats.example
+Note that RDBMS metrics are only relevant if MongooseIM is [configured with an RDBMS backend](../advanced-configuration/database-backends-configuration.md), MAM metrics when [mod_mam is enabled](../modules/mod_mam.md) and MAM flush times when MAM is configured with an RDBMS backend with `async_writer` option (default).
 
-Then, if you configure your MongooseIM to send Exometer reports to that IP address and run it for a while, you should be able to see some interesting charts.
+#### Example graph in Grafana
 
-### Run graphite in Docker - alternative method
+![An example graph in Grafana](example-grafana-graph.png)
 
-If one of these steps doesn't work for you (e.g. they may be incompatible with your Docker on Mac), please try a less advanced alternative.
-It doesn't require a docker machine and will expose Graphite ports on localhost, instead of using a new route.
+This screenshot shows a graph plotting the RDBMS simple query time metric mentioned above.
+The graph is plotted for three nodes with each node having a different prefix: `mongoose.node1`, `mongoose.node2` and `mongoose.node3`.
 
-Start a container with Graphite:
+The queries take metrics for all nodes and all domains (`**` is a wildcard for multiple parts of the metric name) and groups them *per-node*, *per-value-type* - respectively `1`st (indexed from `0`) and `-1`st (last) parts of the metric's name.
```

Comment:
The queries take metrics for all nodes and all domains (`**` is a wildcard for multiple parts of the metric name) and group them *per-node* and *per-value-type* (respectively `1`st and `-1`st part of the metric's name). The parts of the names are indexed from 0.